trainee_template/explanation.amlg

```diff
@@ -307,7 +307,15 @@
 			)
 			(call CalculateResidualsForCase (assoc
 				features features
-				case_values (append context_values action_values)
+				case_values
+					(append
+						context_values
+						;use original case values instead of predicted if computing for an existing case
+						(if ignore_case
+							(retrieve_from_entity ignore_case action_features)
+							action_values
+						)
+					)
```

Comment:
do we need to do anything to be sure these values are in the same order as 'features'?

Reply:
no, features is (append context_features action_features), and action_features are same order as action_values, so it's all ordered consistently